frontend/src/metabase/admin/settings/components/SettingsUpdatesForm/SettingsUpdatesForm.tsx

```diff
@@ -0,0 +1,88 @@
+import cx from "classnames";
+import { c } from "ttag";
+
+import { UpsellHostingUpdates } from "metabase/admin/upsells";
+import { useSetting } from "metabase/common/hooks";
+import CS from "metabase/css/core/index.css";
+import { Flex } from "metabase/ui";
+import type { SettingDefinition, UpdateChannel } from "metabase-types/api";
+
+import { SettingsSetting } from "../SettingsSetting";
+
+import { VersionUpdateNotice } from "./VersionUpdateNotice/VersionUpdateNotice";
+
+const updateChannelSetting = {
```

Comment:
Not worth blocking a merge, but is there a reason this wasn't defined in `selectors.js`?